js/bytex.js

```diff
@@ -463,6 +503,20 @@ module.exports = class bytex extends Exchange {
         return this.parseTickers (response, symbols);
     }
 
+    parseTickers (response, symbols = undefined, params = {}) {
+        const result = {};
+        const keys = Object.keys (response);
+        for (let i = 0; i < keys.length; i++) {
+            const key = keys[i];
+            const ticker = response[key];
+            const marketId = this.safeString (ticker, 'symbol', key);
+            const market = this.safeMarket (marketId, undefined, '-');
+            const symbol = market['symbol'];
+            result[symbol] = this.extend (this.parseTicker (ticker, market), params);
+        }
+        return this.filterByArray (result, 'symbol', symbols);
+    }
+
```

Comment:
This method isn't necessary, there's a `parseTickers` method on the superclass `Exchange.js`